### PR TITLE
Feature/create unit tests

### PR DESCRIPTION
**Pull request by ghost**
@adslaton @turner-martytrujillo I created some unit tests based off of the first two examples to look at the logic of injector. If there are anymore tests, feel free to tell me.

### CODE REVIEW

File: tests/mocha/test.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <script src="../../node_modules/mocha/mocha.js"></script>
+        <script>mocha.setup('bdd')</script> 
+        <script src="https://code.jquery.com/jquery-1.12.3.min.js"></script>
+        <script>
+            window.FAI = {
+                bundleHost: 'bundles/',
+                WebpackAssets: [{'name':'inject.js','chunkNames':['inject']}]
+            };
+        </script>
+        <script src="http://registry.api.cnn.io/assets/js/injector.lite.0.10.0.min.js"></script>
```
Comment (martytheparty) [inline tests/mocha/test.html]: This is awesome, but I think the unit test should run against the src (/src/injector.js) not the injector in the registry.

Comment (ghost) [reply tests/mocha/test.html]: I have changed it

**Comment by adslaton** [comment]
Take a look at the conflicts in:

```
gulpfile.js
package.json
```

**Review by martytheparty — COMMENTED** [review]
I pulled this code down and I ran the tests.  I am seeing this error:

Error loading resource file:///Users/c-expansion-mtrujill/dev/cnnlabs/injector/tests/mocha/bundles/inject.js (203). Details: Error opening /Users/c-expansion-mtrujill/dev/cnnlabs/injector/tests/mocha/bundles/inject.js: No such file or directory

I am seeing that error twice.

**Comment by ghost** [comment]
I have fixed the merge conflict and have uploaded the bundle for the mocha test because it was being ignored in .gitignore.

**Comment by martytheparty** [comment]
I think that the names test.html and test2.html are too vague.

**Comment by martytheparty** [comment]
This all looks good are you planning to add any more unit tests or any CHAI tests?  If you're not planing on anything else I am ready to approve.

**Comment by ghost** [comment]
No, that is all. If I think of more in the future. I will just create a PR for it.

**Comment by martytheparty** [comment]
Approved

**Review by martytheparty — APPROVED** [review]
Thanks for the unit tests!!!

**Comment by adslaton** [comment]
Wonderful start!

**Review by adslaton — APPROVED** [review]

**Pull request merged**